backend/src/libs/packages/server-application/server-app.ts

```diff
@@ -7,13 +7,17 @@ import { type IConfig } from '~/libs/packages/config/config.js';
 import { type IDatabase } from '~/libs/packages/database/database.js';
 import { HttpCode, HttpError } from '~/libs/packages/http/http.js';
 import { type ILogger } from '~/libs/packages/logger/logger.js';
+import { authorization } from '~/libs/plugins/authorization/authorization.plugin.js';
```

Comment:
You need to create an authorization.ts file from which you will reexport authorization.plugin.js.
```suggestion
import { authorization } from '~/libs/plugins/authorization/authorization.js';
```